plugins/@grouparoo/airtable/src/lib/disconnect.ts

```diff
@@ -4,5 +4,5 @@ import { IClient } from "./client/interfaces/iClient";
 export const disconnect: DisconnectPluginAppMethod<IClient> = async ({
   connection,
 }) => {
-  await connection.close();
+  return;
 };
```

Comment:
I mean to say that you can just skip this file entierly if you don't really need to disconnect, and remove it from the initializer too